bigchaindb_driver/transport.py

```diff
@@ -0,0 +1,23 @@
+from .connection import Connection
+from .pool import Pool
+
+
+class Transport:
+
+    def __init__(self, *nodes):
+        self.nodes = nodes
+        self.init_pool(nodes)
+
+    def init_pool(self, nodes):
+        """Initialize the pool of connections."""
+        connections = [Connection(node_url=node) for node in nodes]
+        self.pool = Pool(connections)
+
+    def get_connection(self):
+        """Get a connection from the pool."""
+        return self.pool.get_connection()
+
+    def forward_request(self, method='GET', path='/', json=None):
```

Comment:
Could probably remove these default variables and just use `**kwargs` since `connection.request()` already has these defaults (or the other way around, but it probably makes more sense for these defaults to live in `Connection`).